app/commands/decidim/kids/close_session_managed_minor.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    # A command with all the business logic to close a current impersonation session.
+    class CloseSessionManagedMinor < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # current_user       - The user impersonating a minor
+      # minor_user         - The user to impersonate
```

Comment:
There is a difference between the attribute in the comment and the attribute used in the method: `minor_user` vs `real_user`